lib/curl/curl.nit

```diff
@@ -168,71 +170,106 @@ class CurlHTTPRequest
 			return answer_failure(0, "Unable to extract file name, please specify one")
 		end
 
-		success_response.i_file = new OFile.open(opt_name.to_cstring)
-		if not success_response.i_file.is_valid then
-			success_response.i_file.close
+		success_response.file = new FileWriter.open(opt_name)
+		if not success_response.file.is_writable then
 			return answer_failure(0, "Unable to create associated file")
 		end
 
 		var err_resp = perform
 		if err_resp != null then return err_resp
 
-		var st_code = self.curl.prim_curl.easy_getinfo_long(new CURLInfoLong.response_code)
-		if not st_code == null then success_response.status_code = st_code.response
+		var st_code = self.curl.native.easy_getinfo_long(new CURLInfoLong.response_code)
+		if not st_code == null then success_response.status_code = st_code
 
-		var speed = self.curl.prim_curl.easy_getinfo_double(new CURLInfoDouble.speed_download)
-		if not speed == null then success_response.speed_download = speed.response
+		var speed = self.curl.native.easy_getinfo_double(new CURLInfoDouble.speed_download)
+		if not speed == null then success_response.speed_download = speed
 
-		var size = self.curl.prim_curl.easy_getinfo_double(new CURLInfoDouble.size_download)
-		if not size == null then success_response.size_download = size.response
+		var size = self.curl.native.easy_getinfo_double(new CURLInfoDouble.size_download)
+		if not size == null then success_response.size_download = size
 
-		var time = self.curl.prim_curl.easy_getinfo_double(new CURLInfoDouble.total_time)
-		if not time == null then success_response.total_time = time.response
+		var time = self.curl.native.easy_getinfo_double(new CURLInfoDouble.total_time)
+		if not time == null then success_response.total_time = time
 
-		success_response.i_file.close
+		success_response.file.close
 
 		return success_response
 	end
 end
 
 # CURL Mail Request
-class CurlMailRequest
+#
+# ~~~
+# # Craft mail
+# var mail = new CurlMail("sender@example.org",
+#                            to=["to@example.org"], cc=["bob@example.org"])
+#
+# mail.headers_body["Content-Type:"] = """text/html; charset="UTF-8""""
+# mail.headers_body["Content-Transfer-Encoding:"] = "quoted-printable"
+#
+# mail.body = "<h1>Here you can write HTML stuff.</h1>"
+# mail.subject = "Hello From My Nit Program"
+#
+# # Set mail server
+# var error = mail.set_outgoing_server("smtps://smtp.example.org:465",
+#                                      "user@example.org", "mypassword")
+# if error != null then
+#     print "Mail Server Error: {error}"
+#     exit 0
```

Comment:
This test does not work, the server does not exists, so `exit 1` make nitunit report it as an error.